pkgs/applications/version-management/git-and-tools/hut/default.nix

```diff
@@ -32,8 +31,6 @@ buildGoModule {
     make $makeFlags install
   '';
 
-  passthru.updateScript = unstableGitUpdater { };
-
```

Comment:
Will it still be automatically updated without `passthru.updateScript`?

Reply:
I would hope so. If it is properly tagged, why shouldn't it?

Reply:
nix-update can update from github just fine. No need to add extra sugar.

Reply:
This package lives on sourcehut, but should still be possible.